data/hai/hai reveal 4 middle.txt

```diff
@@ -1108,7 +1108,7 @@ mission "Hai Reveal [A10] Demanding Satisfaction"
 			`	"You haven't the authority to do that," he responds, laughing.`
 			`	"Besides..." you see one of the other Unfettered to the right of the leader begin to step out. "You would ha..." Teeneep cuts the leader off as she draws a throwing knife and launches it. It zips through the air, and spears the shoulder of the Unfettered who began to move, embedding itself in the wall behind and pinning them in place.`
 			`	"I would have to what, defeat your right hand first?" She takes a step back and smirks at him. "Now, will you face a woman half your size?" From her expression, you're unsure if she genuinely has that much faith in her abilities, or if this is a bluff.`
-			`	The Unfettered leader pauses, then smiles. "There is no need. Even on our planets, I've rarely met a Hai half as brave as you. You have the courage to turn an act into action. You are a True Hai, and we protect our own. You have vouched for the value of the lives of these Hai, so I say now we are willing to send our fleets to their aid. I would come myself."`
+			`	The Unfettered leader's left hand was already twitching towards where an ancient-looking dagger hung on his belt, instinctively, as he was readying to defend himself from the potential assault, but he pauses. You notice the dagger was already changing shape and growing wider, but it returns to its previous state as he starts smiling and puts his hand away. "There is no need. Even on our planets, I've rarely met a Hai half as brave as you. You have the courage to turn an act into action. You are a True Hai, and we protect our own. You have vouched for the value of the lives of these Hai, so I say now we are willing to send our fleets to their aid. I would come myself."`
```

Comment:
What tense is this supposed to be in? This sentence also seems rather long and difficult to read.

Reply:
Tense? I'm really not good at english, I have no idea